src/components/MyPage/MoveBoard.tsx

```diff
@@ -53,8 +64,8 @@ const MoveBoard = () => {
       </StHeader>
 
       <StHoverShadow className={hoverTarget === `moveBoard${index}` ? 'view' : ''} />
-      <StEdit className={hoverTarget === `moveBoard${index}` ? 'view' : ''}>
-        <ICEdit width="16px" height="16px" />
+      <StEdit className={hoverTarget === `moveBoard${index}` ? 'view' : ''} onClick={(e) => handleOnClick(e)}>
```

Comment:
요거 e 안넘겨주고 
```suggestion ts
onClick={handleOnClick}
```
하면 되지아나?!